File: src/nomanssave/w.java
package nomanssave;

class w implements Runnable {
   // $FF: synthetic field
   private final boolean ba;

   w(boolean var1) {
      this.ba = var1;
   }

   public void run() {
      try {
         Application.g(new Application(this.ba, (Application)null));
         Application.h(Application.H()).setVisible(true);
      } catch (Exception var2) {
         var2.printStackTrace();
         System.exit(1);
      }

   }
}

File: NMSSaveEditor/nomanssave/lower/w.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{
public class w : Runnable {
   public bool ba;

   public w(bool var1) {
      this.ba = var1;
   }

   public void run() {
      try {
         Application.g(new Application(this.ba, (Application)null));
         Application.h(Application.H()).Show();
      } catch (Exception var2) {
         var2.printStackTrace();
         Environment.Exit(1);
      }

   }
}

}
